dist/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
just all of dist/*

Reply:
So the way we have it set up, we're serving the page from Github. So we need this checked in. Two other potential options:
 - we serve the compiled script from somewhere else (e.g. Heroku)
 - we set up github pages configuration so it's serving a branch other than master and then have the deploy process work like this:
   - merge into master
   - checkout other branch and merge in master
   - build and the assets and push

Reply:
oh right. eh the second process feels janky. The idea of having a 'prod' and 'master' branch feel off.

Reply:
yeah. I say we go with checking in the scripts for now and then decide to deploy elsewhere if we actually keep using this.